src/borg/archiver/_common.py

```diff
@@ -423,14 +423,14 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True, ol
             metavar="Nd",
             type=relative_time_marker_validator,
             dest="older",
-            help="consider archives older than N-days ago",
+            help="consider archives older than N [days/months] ago",
         )
         group.add_argument(
             "--newer",
             metavar="Nd",
             type=relative_time_marker_validator,
             dest="newer",
-            help="consider archives between now and N-days ago",
+            help="consider archives between now and N [days/months] ago",
```

Comment:
maybe the metavar should be "TIMESPAN" or so, not "Nd".

the help text then could refer to TIMESPAN (and maybe 7d and 12m as an example).